### PR TITLE
Set up timer to automatically refresh data

### PR DESCRIPTION
**Pull request by kq-li**

### CODE REVIEW

**Comment by kq-li** [comment]
yep, tried a few queries and they all went through fine -- we should write robust test suites once things are settled

File: app.py [hunk 1]
```diff
@@ -3,22 +3,21 @@
 from flask import Flask
 from flask_graphql import GraphQLView
 from graphene import Schema
+import data
```
Comment (young-k) [inline app.py]: alphabetize imports?

File: data.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from datetime import datetime
+import threading
+import constants
+import scraper
```
Comment (young-k) [inline data.py]: whoops, same thing here?

File: app.py (hunk 1 above)
Comment (kq-li) [reply app.py]: they're separated into subcategories of default python modules, 3rd party modules, and local modules and alphabetized within subcategories i think

**Comment by kq-li** [comment]
@young-k thanks i try LOL result of multiple redesigns i guess

**Pull request merged**